src/FieldEncryptProcessEntities.php

```diff
@@ -0,0 +1,302 @@
+<?php
+/**
+ * @file Contains \Drupal\field_encrypt\FieldEncryptProcessEntities
+ */
+
+namespace Drupal\field_encrypt;
+
+use Drupal\Core\Entity\EntityManager;
+use Drupal\Core\Entity\Query\QueryFactory;
+use Drupal\encrypt\EncryptService;
+
+/**
+ *
+ */
+class FieldEncryptProcessEntities {
+
+  /**
+   * Filter (blacklist) field types that don't work with encryption.
+   *
+   * This list is mostly to avoid problems, but we shouldn't let people encrypt
+   * these fields since we know it wouldn't work.
+   */
+  public $blacklist_fields = [
+    'image',
+    'entity_reference',
+    'datetime',
+    'boolean',
+    'integer',
+    'decimal',
```

Comment:
When you encrypt a number with the simple key in the `key` module, you get a long string which isn't compatible with the sql table storage.